packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js

```diff
@@ -129,6 +129,49 @@ export class MyAccountMyWishlist extends PureComponent {
 
     renderProduct([id, product]) {
         const { isEditingActive, loadingItemsMap, setIsQtyUpdateInProgress } = this.props;
+        const {
+            wishlist = {}
+        } = product;
+
+        // assign wishlist.options to a new list
+        // remove the first part of the value string that indicated quantity
+        // append with a value from buy_request instead
+        // get the value using index
+        if (wishlist.options.length >= 1) {
+            // the order of items in wishlist.options always corresponds with the order of items
+            // in buy_request.
+            // we're using regex instead of JSON parse bc we need to preserve the order of the data
+            // objects are inherently unordered
+            const { buy_request } = wishlist;
+            const selections = buy_request.match(/bundle_option(.*)bundle/g)[0].match(/\d+/g);
+            const parsedSelections = [];
+            selections.map((e, i) => {
+                if ((i + 1) % 2 === 0) {
+                    parsedSelections.push({
+                        [selections[i - 1]]: e
+                    });
+                }
+
+                return '';
+            });
+            const quantities = buy_request.match(/bundle_option_qty(.*)/g)[0].match(/\d+/g);
```

Comment:
If the regex fails, the `match` function will return null and this code will crash.